cms/envs/common.py

```diff
@@ -546,24 +546,25 @@
 PIPELINE_JS_COMPRESSOR = None
 
 STATICFILES_IGNORE_PATTERNS = (
-    "*.py",
-    "*.pyc"
+    '*.py',
+    '*.pyc'  # <--- Something doesn't seem right.  Missing comma?
```

Comment:
Good catch. This looks like it will be implicitly concatenated with the following string `"sass/*.scss"` (unless the inline comments change the string behavior).

Let me see if they have this code upstream. If they do, we should consider submitting this patch to edX. Remind me to discuss this tomorrow.